app/block_address.go

```diff
@@ -32,5 +40,13 @@ func (bad BlockAddressesDecorator) AnteHandle(ctx sdk.Context, tx sdk.Tx, simula
 			}
 		}
 	}
+	admin := bad.getParams(ctx).CronosAdmin
```

Comment:
I mean, it should happen in check tx, because msg handler will do the check anyway.